guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -93,12 +93,14 @@ ifdef::foreman-el,katello,satellite[]
 ----
 endif::[]
 ifdef::satellite[]
+----
+# dnf module enable {dnf-modules} postgresql:12 ruby:2.7
+----
++
+If `postgresql:12` and `ruby:2.7` fail to get enabled, refer xref:Troubleshooting_{context}[Troubleshooting Guide].
```

Comment:
Perhaps keep this generic and avoid duplicating the exact versions?
```suggestion
If modules fail to get enabled, refer xref:Troubleshooting_{context}[Troubleshooting Guide].
```

Also, perhaps we can link directly to a chapter within the troubleshooting guide?

Reply:
Addressed :+1: